tests/test_browser.py

```diff
@@ -95,3 +106,29 @@ def test_save_driver_logs(self):
         for log_type in log_types:
             expected_file = os.path.join(tempdir_path, 'js_page_{}.log'.format(log_type))
             self.assertTrue(os.path.isfile(expected_file))
+
+    def test_profile_error(self):
+        """
+        If there is a WebDriverException when instantiating the driver,
+         it should be tried again.
+        """
+        patcher = patch('bok_choy.browser._local_browser_class')
+        patch_object = patcher.start()
+        patch_object.side_effect = WebDriverException()
```

Comment:
A small improvement to the mock would be to have it raise the error with parameters.
From http://selenium-python.readthedocs.org/api.html#selenium.common.exceptions.WebDriverException maybe something like:
`patch_object.side_effect = WebDriverException(msg='that did not work', screen=None, stacktrace=None)`